travel/travel.py

```diff
@@ -25,6 +25,17 @@
 from .res_config import get_basic_passenger_limit
 
 
+def _get_travel_states(self, cr, uid, ids=None, context=None):
```

Comment:
I mean the bug in ORM that doesn't resolve this correctly, as mentioned by @bwrsandman. This is only a workaround as ORM is not populating correctly the selection list.